n3fit/src/n3fit/scripts/evolven3fit.py

```diff
@@ -82,6 +82,13 @@ def construct_evolven3fit_parser(subparsers):
     parser.add_argument(
         "-d", "--dump", type=pathlib.Path, default=None, help="Path where the EKO is dumped"
     )
+    parser.add_argument(
+        "-n",
+        "--ncores",
+        type=int,
+        default=1,
+        help="Specify the number of cores to parallelize evolution",
+    )
```

Comment:
Careful with the naming, there _used_ to be a `-n` flag already.

Reply:
But they live in different "parser", no? In the same way as `-p` for example (which appears at the top level `evolvenefit`, `produce_eko`, and `produce_eko_photon`).

Reply:
ah, not sure tbh, I just remember having used it.
But then can't you reuse that `-n`?

Reply:
In principle we could. I am just not sure if there is a situation in which these two have to be different.

But actually given that this is a really temporary solution, let me do so.

Reply:
> I am just not sure if there is a situation in which these two have to be different.

No, the existing one is used for the generation of the eko. But the generation and the evolution are always running separately so no problem

Reply:
Done in [a1f5abd](https://github.com/NNPDF/nnpdf/pull/2168/commits/a1f5abd0ed5a674369caa0682fa19e82402c6d4f).